src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -407,4 +413,12 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
     }
 
     // ################## ONLY FOR LOCAL TESTING PURPOSE -- END ##################
+
+    onChildActivate(ref: StudentQuestionsComponent) {
```

Comment:
please add this code earlier in the file before the section ONLY FOR LOCAL TESTING PURPOSE
please also add some documentation here

Reply:
done